packages/teams-js/src/public/pages.ts

```diff
@@ -66,12 +74,16 @@ export function tabsNavigateToTabHelper(apiVersionTag: string, tabInstance: TabI
   });
 }
 
-export function returnFocusHelper(apiVersionTag: string, navigateForward?: boolean): void {
+export function returnFocusHelper(
+  apiVersionTag: string,
+  navigateForward?: boolean,
+  returnFocusActionItem?: ReturnFocusActionItem,
```

Comment:
We might need to set up a few minutes for me to ask some questions about this PR. I see that you're adding a new optional parameter here for returnFocusActionitem to a public function that is available to all developers, but you're marking the interface itself in `interfaces.ts` as hidden and internal. This seems wrong-ish to me, but I want to ask more questions